sources/packages/backend/apps/queue-consumers/src/processors/schedulers/sfas-integration/_tests_/sims-to-sfas-integration.scheduler.e2e-spec.ts

```diff
@@ -174,7 +686,7 @@ describe(describeProcessorRootTest(QueueNames.SIMSToSFASIntegration), () => {
    * @returns created student.
    */
   async function createStudentWithExpectedData(
-    expectedUpdatedDate: Date,
+    expectedUpdatedDate?: Date,
```

Comment:
as per the comment https://github.com/bcgov/SIMS/pull/3895/files#r1844654655 we don't have to make it optional.

Reply:
Keeping it optional as discussed.